playwright/utils.ts

```diff
@@ -0,0 +1,291 @@
+import { expect, Page, ElementHandle, Locator } from "@playwright/test";
+
+export const checkIfOnDashboard = async (page: Page) => {
+    await page.waitForTimeout(1000);
```

Comment:
What is the reason for adding manual timeouts (and in other places in this PR)? If the website is slow, perhaps setting a higher global timeout is better

Reply:
Sometimes the tests glitch in some areas for example when switching between pages or waiting for dropdown options to load so I just added it in the relevant places

Reply:
`waitForTimeout` is discouraged: https://playwright.dev/docs/api/class-page#page-wait-for-timeout

There are better options such as waiting on network events or using more precise selectors. You can also increase the timeout on individual selectors when you know it is going to be a slow event: https://playwright.dev/docs/test-timeouts#introduction

Reply:
It is causing issues for me to switch to the other approach - let's discuss this

Reply:
Added to tech debt if there is time as an improvement